examples/post_training_quantization/openvino/yolov8/main.py

```diff
@@ -0,0 +1,174 @@
+        return input_tensor
+    
+    quantization_dataset = nncf.Dataset(data_loader, transform_fn)
+    
+    quantized_model = nncf.quantize(
+                            model, 
+                            quantization_dataset,
+                            preset=nncf.QuantizationPreset.MIXED,
+                            ignored_scope=nncf.IgnoredScope(
+                                types=["Multiply", "Subtract", "Sigmoid"],  # ignore operations
+                                names=["/model.22/dfl/conv/Conv",           # in the post-processing subgraph
+                                    "/model.22/Add",
+                                    "/model.22/Add_1",
+                                    "/model.22/Add_2",
+                                    "/model.22/Add_3",
+                                    "/model.22/Add_4",
+                                    "/model.22/Add_5",
+                                    "/model.22/Add_6",
+                                    "/model.22/Add_7",
+                                    "/model.22/Add_8",
+                                    "/model.22/Add_9",
+                                    "/model.22/Add_10"]
+                            ))
+    return quantized_model
+
+    
+def main():
+    MODEL_NAME = "yolov8n"
+    
+    model = YOLO(f"{MODEL_NAME}.pt")
+    args = get_config(config=DEFAULT_CONFIG)
+    args.data = "coco128.yaml"
+
+    # Prepare validation dataset and helper
+    validator, data_loader = prepare_validation(model, args)
+    
+    # Convert to OpenVINO model
+    ov_model, ov_model_path = prepare_openvino_model(model, MODEL_NAME)
+    
+    # Quantize mode in OpenVINO representation
+    quantized_model = quantize(ov_model, data_loader, validator)
+    quantized_model_path = Path(f"{MODEL_NAME}_openvino_model/{MODEL_NAME}_quantized.xml")
+    ov.serialize(quantized_model, str(quantized_model_path))
+    
+    # Validate FP32 model    
+    fp_stats, total_images, total_objects = validate(ov_model, tqdm(data_loader), validator)
+    print("Floating-point model validation results:")
+    print_statistics(fp_stats, total_images, total_objects)
+    
+    # Validate quantized model
+    q_stats, total_images, total_objects = validate(quantized_model, tqdm(data_loader), validator)
+    print("Quantized model validation results:")
+    print_statistics(q_stats, total_images, total_objects)
+    
+    # Benchmark performance of FP32 model
+    fp_model_perf = benchmark_performance(ov_model_path, args)
+    print(f"Floating-point model performance: {fp_model_perf} FPS")
+    
+    #Benchmark performance of quantized model
+    quantized_model_perf = benchmark_performance(quantized_model_path, args)
+    print(f"Quantized model performance: {quantized_model_perf} FPS")
+    
+    return fp_stats["metrics/mAP50-95(B)"], q_stats["metrics/mAP50-95(B)"], \
+        fp_model_perf, quantized_model_perf
+
+if __name__ == "__main__":
```

Comment:
Do we use __main__  in the example?

Reply:
The plan is to import the main from the example and use it in the test to validate this on regular basis.